src/gateways/createArchiveLoader.ts

```diff
@@ -27,13 +28,19 @@ export default (url: string) => {
   };
 
   const extrasCache = new Map();
-  const getBookIdsForPage: (pageId: string) => Promise<string[]> = (pageId) => {
+  const getBookIdsForPage: (pageId: string) => Promise<Array<{id: string, bookVersion: string | undefined}>> =
```

Comment:
I would write this type like this:
`Promise<{id: string, bookVersion?: string}[]>`
 because it is shorter

Reply:
Lint modified it, that's why it is written this way.

Reply:
`Array<{id: string, bookVersion: string | undefined}>` is the correct way